src/spaces/p3.js

```diff
@@ -1,23 +1,17 @@
-import Color from "./srgb.js";
+import RGBColorSpace from "../rgbspace.js";
+import P3Linear from "./p3-linear.js";
+import sRGB from "./srgb.js";
 
-Color.defineSpace({
-	inherits: "srgb",
+export default RGBColorSpace.create({
 	id: "p3",
 	name: "P3",
+	base: P3Linear,
 	cssId: "display-p3",
 	// Gamma correction is the same as sRGB
 	// convert an array of display-p3 values to CIE XYZ
```

Comment:
Due to the matrix moving to p3-linear, all but the first line in this comment can be deleted